balances/src/lib.rs

```diff
@@ -99,6 +99,14 @@ decl_error! {
 }
 
 impl<T: Config> Module<T> {
+	pub fn get_balances(account: &T::AccountId) -> Vec<(CommunityIdentifier, BalanceType)> {
```

Comment:
it's not ok to have `get_balance` and `balance` functions here. both are getters. Please rename to `get_all_balances`

Reply:
or maybe `get_all_balances_demurraged`. see below